### PR TITLE
Show votes when an event is archived.

### PR DESCRIPTION
**Pull request by akatsoulas**
* Show questions in a random order
* Admins can see votes

### CODE REVIEW

File: moderator/moderate/views.py [hunk 1]
```diff
@@ -130,7 +129,10 @@ def show_event(request, e_slug, q_id=None):
     questions_q = Question.objects.filter(event=event).annotate(
         vote_count=Count("votes")
     )
-    questions = questions_q.order_by("-vote_count")
+    if user.is_superuser:
```
Comment (LeoMcA) [inline moderator/moderate/views.py:132]: Should this be `user.userprofile.is_admin` instead?

File: moderator/moderate/models.py [hunk 2]
```diff
@@ -30,7 +29,7 @@ def save(self, *args, **kwargs):
 
     @property
     def is_admin(self):
-        return self.user.groups.filter(name='Admin').exists()
+        return self.user.groups.filter(name='Admin').exists() | self.user.is_superuser
```
Comment (LeoMcA) [inline moderator/moderate/models.py:32]: Would it be more pythonic to use `or` rather than `|` here?

**Review by LeoMcA — APPROVED** [review]
r+wc

File: moderator/moderate/views.py (hunk 1 above)
Comment (LeoMcA) [reply moderator/moderate/views.py:132]: Actually should be this be `event.archived or user.userprofile.is_admin` to reflect the logic expressed in the issue?
> When an events gets archived, the questions are ordered based on the vote count

**Review by The-smooth-operator — APPROVED** [review]
Agreed with Leo's comments. We can merge this for testing in staging however

**Comment by The-smooth-operator** [comment]
Mmmmmm Travis fails because $AWS_ACCOUNT_ID is null, however it is defined and didn't change for a while.

**Comment by The-smooth-operator** [comment]
> Mmmmmm Travis fails because $AWS_ACCOUNT_ID is null, however it is defined and didn't change for a while.

ok NVM I think the problem is that you made the PR with your fork as origin, so it doesn't have the right secrets as is expected.
I'll take a note for fixing this problem.

**Pull request merged**